site/build.js

```diff
@@ -52,7 +52,16 @@ metalSmith(__dirname)
     .use(function (files, metalsmith, next) {
         var metadata = metalsmith.metadata();
 
-        var assertionsByType = {};
+        // Make sure that the most important types are listed first and in this order:
+        var assertionsByType = {
+            any: [],
+            array: [],
+            boolean: [],
+            function: [],
+            number: [],
+            object: [],
+            string: []
+        };
```

Comment:
I actually agree with you that it will give a better result with a predefined order for the types. I'll merge :-)